app-dicts/myspell-en/myspell-en-20181101.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+MY_PV="${PV:0:4}-${PV:4:2}.${PV:6:2}"
+
+inherit myspell-r2
+
+DESCRIPTION="English dictionaries for myspell/hunspell"
+HOMEPAGE="https://extensions.libreoffice.org/extensions/english-dictionaries"
+SRC_URI="https://extensions.libreoffice.org/extensions/english-dictionaries/${MY_PV}/@@download/file/dict-en-${PV}.oxt"
+
+LICENSE="BSD MIT LGPL-3+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
+IUSE=""
+
+PLOCALES=( "en" "en-AU" "en-CA" "en-GB" "en-US" "en-ZA" )
+IUSE+="${PLOCALES[@]/#/l10n_}"
+
+src_prepare() {
+	if use "l10n_en-GB" || use "l10n_en"; then
+		MYSPELL_HYPH=( "hyph_en_GB.dic" )
+	fi
+	if use "l10n_en-US" || use "l10n_en"; then
+		MYSPELL_THES=( 
+			"th_en_US_v2.dat" 
+			"th_en_US_v2.idx"
+		)
+	fi
+
+	MYSPELL_DICT=( )
+	for lang in "${PLOCALES[@]/%en}"; do	
```

Comment:
Hmm, doesn't this mean you'd get one loop iteration with `lang=""`?

Reply:
No, if that was happening you would get an error on adding a file .dic

Reply:
Hmm, can you explain to me what the `/%en` substitution does here?

Reply:
It removes the first item on from the array, because no such language actually exists so you cannot add en.dic without throwing an error.

Reply:
That's not throwing any error for me

Reply:
ok so i'll just add another if statement, but this all seems messy to me.. but it works while not introducing an "" element